packages/cli/src/commands/deploy/helper.ts

```diff
@@ -77,18 +87,30 @@ export const upsertFieldPlugin: UpsertFieldPluginFunc = async (args) => {
     // update flow
     const mode = skipPrompts ? 'update' : await selectUpsertMode()
     if (mode === 'update') {
+      if (manifest?.options !== undefined) {
+        const confirmed = await confirmOptionsUpdate()
```

Comment:
Good point @eunjae-lee. 🙌 
I totally agree with considering it as `confirmed` by default when the `skipPrompts` parameter is informed.

It will avoid breaking our CI/CD in the future as well.

I've made the change.
Thanks for the suggestion.